eventually-postgres/src/store.rs

```diff
@@ -136,9 +176,9 @@ impl EventStoreBuilderMigrated {
     pub async fn build<Id, Event>(
         &self,
         type_name: &'static str,
-    ) -> std::result::Result<EventStore<Id, Event>, tokio_postgres::Error> {
+    ) -> std::result::Result<EventStore<Id, Event, Tls>, RunError<tokio_postgres::Error>> {
```

Comment:
What do you think about defining a type alias?

```rust
pub type PoolResult<T> = std::result::Result<T, RunError<tokio_postgres::Error>>
```

So you can change this statement to:
```rust
pub async fn build<Id, Event>(
    &self,
    type_name: &'static str,
) -> PoolResult<EventStore<Id, Event, Tls>>
```

Same goes for this: https://github.com/eventually-rs/eventually-rs/pull/137/files#diff-d419ac38248513a9a51b12671bf15f4dbf69d2b16002da96ae05f765edad9657R204-R207